metrics_collector/volume_collector.go

```diff
@@ -251,6 +265,14 @@ func getVolumeRobustnessValue(v *longhorn.Volume) int {
 	return robustnessValue
 }
 
+func getLastBackupAtValue(v *longhorn.Volume) int64 {
+	lastBackupAt, err := time.Parse(time.DateTime, v.Status.LastBackupAt)
+	if err != nil {
+		return 0
+	}
```

Comment:
Suggest to inputting just the lastBackupAt rather than the entire Volume object since there is no use for other fields here.
1. If the value is "", return 0
1. If the time pasering fails, print a trace log so the error is not swollowed, and then then return 0.